tg/controllers.py

```diff
@@ -635,6 +635,38 @@ def delete_chat(self) -> None:
 
         self.present_info("Chat was deleted")
 
+    @bind(chat_handler, ["dm"])
+    def delete_messages_in_chat(self) -> None:
+        """Delete last 100 user messages in the group"""
```

Comment:
I'm not sure if this deserves an own key binding. If it does it should probably accept a count and default to deleting one message.